tests/test-abs.sh

```diff
@@ -0,0 +1,36 @@
+#! /bin/bash
```

Comment:
BTW i like the idea of having external scripts to test. I will enhance this in order to add more "integration" tests that run abs scripts and compare expected vs real output and status code. Thanks for the inspiration.